bin/consensus-tests.rs

```diff
@@ -837,10 +839,6 @@ where
 
     let mut out = RunResults::default();
     for (test_name, test) in j {
-        if !test_names.is_empty() && !test_names.contains(&test_name) {
```

Comment:
Yeah, I guess shared ownership is the way to go. As otherwise there's no way to show the borrow checker that `test_names` is alive for the duration of tokio coroutines, is there?

Reply:
Outside using scoped tasks (not present in tokio), no, there isn't.